libraries/context_comparator/virl.yaml

```diff
@@ -0,0 +1,30 @@
+testbed:
+    name: 'virl'
+
+devices:
+    nx-osv-1:
+        type: "NX-OSv 9000"
+        os: "nxos"
+        alias: uut
+        credentials:
+            default:
+                username: root
+                password: D_Vay!_10&
```

Comment:
isn't the old username and password admin admin?

Reply:
done in ce1d1526346665bf797effb7b372a5030d2f9bfd